vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -163,52 +169,52 @@ You will need to restart VS Code after these changes. If PowerShell is still not
      * @remarks Some users have reported not having powershell.exe or having execution policy that fails property evaluation functions in powershell install scripts.
      * We use this function to throw better errors if powershell is not configured correctly.
      */
-    private async verifyPowershellCanRun(installContext : IDotnetInstallationContext, installId : DotnetInstall) : Promise<string>
+    private async verifyPowershellCanRun(installContext: IDotnetInstallationContext, installId: DotnetInstall): Promise<string>
     {
         let knownError = false;
         let error = null;
         let command = null;
 
         const possibleCommands =
-        [
-            CommandExecutor.makeCommand(`powershell.exe`, []),
-            CommandExecutor.makeCommand(`%SystemRoot%\\System32\\WindowsPowerShell\\v1.0\\powershell.exe`, []),
-            CommandExecutor.makeCommand(`pwsh`, []),
-            CommandExecutor.makeCommand(`powershell`, []),
-            CommandExecutor.makeCommand(`pwsh.exe`, [])
-        ];
+            [
+                CommandExecutor.makeCommand(`powershell.exe`, []),
+                CommandExecutor.makeCommand(`%SystemRoot%\\System32\\WindowsPowerShell\\v1.0\\powershell.exe`, []),
+                CommandExecutor.makeCommand(`pwsh`, []),
+                CommandExecutor.makeCommand(`powershell`, []),
+                CommandExecutor.makeCommand(`pwsh.exe`, [])
+            ];
 
         try
         {
             // Check if PowerShell exists and is on the path.
             command = await new CommandExecutor(this.workerContext, this.utilityContext).tryFindWorkingCommand(possibleCommands);
-            if(!command)
+            if (!command)
             {
                 knownError = true;
                 const err = Error(this.noPowershellError);
                 error = err;
             }
 
             // Check Execution Policy
-            const execPolicyOutput = cp.spawnSync(command!.commandRoot, [`-command`, `$ExecutionContext.SessionState.LanguageMode`], {cwd : path.resolve(__dirname), shell: true});
+            const execPolicyOutput = cp.spawnSync(command!.commandRoot, [`-command`, `$ExecutionContext.SessionState.LanguageMode`], { cwd: path.resolve(__dirname), shell: true });

```

Comment:
I'd rename this variable, to prevent the confusion that executionpolicy and languagemode are different concepts.

Reply:
That's a great point, thank you for that suggestion.